content/en/blog/_posts/2021-11-12-are-you-ready-for-dockershim-removal/index.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: blog
+title: "Dockershim removal is coming. Are you ready?"
+date: 2021-11-12
+slug: are-you-ready-for-dockershim-removal
+---
+
+**Author:** Sergey Kanzhelev, Google. With reviews from Davanum Srinivas, Elana Hashman, Noah Kantrowitz, Rey Lejano.
+
+Last year we announced that Dockershim is being deprecated: [Dockershim Deprecation FAQ](https://kubernetes.io/blog/2020/12/02/dockershim-faq/).
+Our current plan is to remove the dockershim from Kubernetes codebase soon.
+We are looking for feedback from you whether you are ready for dockershim
+removal and to ensure that you are ready when the time comes.
+**Please fill out this survey: https://forms.gle/svCJmhvTv78jGdSx8**.
+
+Dockershim component that enables Docker as a Kubernetes container runtime is
+being deprecated in favor of runtimes that directly use the [Container Runtime Interface](/blog/2016/12/container-runtime-interface-cri-in-kubernetes/)
+created for Kubernetes. Many Kubernetes users have migrated to
+other container runtimes without problems. However we see that dockershim is
+still very popular. You may see some public numbers in recent [Container Report](https://www.datadoghq.com/container-report/#8) from DataDog.
+Some Kubernetes hosting vendors just recently enabled other runtimes support
+(especially for Windows nodes). And we know that many third party tools vendors
+are still not ready: [migrating telemetry and security agents](/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents/#telemetry-and-security-agent-vendors).
+
+At this point we believe that there is a feature parity between Docker and other
+runtimes, many customers running production workload with other runtimes, and
+[migration documentation](/docs/tasks/administer-cluster/migrating-from-dockershim/) is clear.
+The plan of record today is that dockershim will be removed in version 1.24.
+Development of this version will start in December and the removal of dockershim
+is an irreversible operation that will happen at the beginning of the 1.24
+release cycle.
```

Comment:
```suggestion
At this point, we believe that there is feature parity between Docker and the
other runtimes. Many end-users have used our [migration guide](/docs/tasks/administer-cluster/migrating-from-dockershim/)
and are running production workload using these different runtimes. The plan of
record today is that dockershim will be removed in version 1.24, slated for 
release around April of next year. For those developing or running alpha and
beta versions, dockershim will be removed in December at the beginning of the
1.24 release development cycle.
```

Reworded a bit to try and make it a little more clear on which audience will be impacted when with its removal.